pkg/v6/configmap/desired.go

```diff
@@ -34,7 +38,8 @@ type IngressControllerGlobal struct {
 }
 
 type IngressControllerGlobalController struct {
-	Replicas int `json:"replicas"`
+	Replicas         int  `json:"replicas"`
+	UseProxyProtocol bool `json:"useProxyProtocol"`
```

Comment:
Currently this is set by overwriting the configmap in aws-operator.

https://github.com/giantswarm/aws-operator/blob/1b16539b73cde4923aad85ddd8640cfcc8371e90/service/controller/v15/templates/cloudconfig/ingress_controller_config_map.go#L13

Now its configurable via the chart.